kafka-connect-s3/docs/configuration_options.rst

```diff
@@ -135,6 +135,12 @@ Partitioner
   * Importance: high
   * Dependents: ``partition.field.name``, ``partition.duration.ms``, ``path.format``, ``locale``, ``timezone``, ``schema.generator.class``
 
+``schema.generator.class``
```

Comment:
Shall we hide this since we're removing it anyway? It is technically accurate in this version, but not useful.